### PR TITLE
Renaming of commands (aesthetics)

### PR DESCRIPTION
**Pull request by lxxxvi**
As requested, all commands have been renamed according to [your specification](https://github.com/mrdoob/three.js/pull/7337#issuecomment-150024472).

```
editor/js/Command.js
editor/js/commands/AddObjectCommand.js
editor/js/commands/AddScriptCommand.js
...
```

### CODE REVIEW

File: editor/docs/Implementing additional commands for undo-redo.md [hunk 1]
```diff
@@ -26,12 +26,12 @@ Every command needs a constructor. In the constructor
...
-CmdXXX = function () {
+DoSomethingCommand = function () {
 
-	Cmd.call( this ); // Required: Call default constructor
+	Command.call( this ); // Required: Call default constructor
 
-	this.type = 'CmdXXX';            // Required: has to match the object-name!
-	this.name = 'Set/Do/Update XXX'; // Required: description of the command, used in Sidebar.History
+	this.type = 'DoSomethingCommand';            // Required: has to match the object-name!
+	this.name = 'Set/Do/Update DoSomething'; // Required: description of the command, used in Sidebar.History
```
Comment (mrdoob) [inline editor/docs/Implementing additional commands for undo-redo.md]: Should be 'Set/Do/Update Something'?

File: editor/docs/Writing unit tests for undo-redo commands.md [hunk 2]
```diff
@@ -16,19 +16,19 @@ Each of the listed steps will now be described in detail.
...
 Within the file, go to the `<!-- command object classes -->` and include the new command:
 
-```javascript
+```html
 // <!-- command object classes -->
 //...
-<script src="../../editor/js/CmdSetUuid.js"></script>
-<script src="../../editor/js/CmdSetValue.js"></script>
-<script src="../../editor/js/CmdXXX.js"></script>         // add this line
+<script src="../../editor/js/AddScriptCommand.js"></script>
```
Comment (mrdoob) [inline editor/docs/Writing unit tests for undo-redo commands.md]: "../../editor/js/commands/AddScriptCommand.js"?

File: editor/js/Command.js [hunk 3]
```diff
@@ -9,7 +9,7 @@
  * @constructor
  */
 
-Cmd = function ( editorRef ) {
+Command = function ( editorRef ) {
```
Comment (mrdoob) [inline editor/js/Command.js]: You should use var for these by the way...

```javascript
var Command = function ( editorRef ) {
```

**Comment by mrdoob** [comment]
Thanks for doing the change. I know it's a bit of a pain...

**Comment by lxxxvi** [comment]
Will change the small things you mentioned. Thanks for reviewing.

**Comment by lxxxvi** [comment]
[...done](https://github.com/lxxxvi/three.js/commit/ec53b8e4f5473c20d45ca5f756e75d1251b6d22a).

**Comment by mrdoob** [comment]
Great!

**Comment by mrdoob** [comment]
Many many thanks!

**Pull request merged**